### PR TITLE
AZP: Rain support

### PR DESCRIPTION
**Pull request by avildema**
Rain support:
* change label from sputnik to ucx_roce
* change interference script for rain switch
* Use analyzer from mtt repo

### CODE REVIEW

File: buildlib/az-network-corrupter.sh [hunk 1]
```diff
@@ -3,17 +3,19 @@
...
 
-manager_script=/hpc/noarch/git_projects/swx_infrastructure/clusters/bin/manage_host_ports.sh
+manager_script_dir=/hpc/noarch/git_projects/hpc-mtt-conf/scripts
+manager_script=${manager_script_dir}/switch_port_on_off.py
 
 if [ "x$reset" = "xyes" ]; then
-    echo "Resetting interface on $(hostname)..."
-    ${manager_script} "$(hostname)" "bond-up"
+    echo "Resetting interface on $(hostname) interface ..."
+    python ${manager_script} -d ${manager_script_dir}/hosts --host $(hostname) -a on -i ${interface}
```
Comment (yosefe) [inline buildlib/az-network-corrupter.sh:18]: why need python? IMO the script should be executable

File: buildlib/pr/io_demo/az-stage-io-demo.yaml [hunk 2]
```diff
@@ -59,7 +60,7 @@ steps:
   timeoutInMinutes: 10
 
 - bash: |
-    python $(workspace)/buildlib/pr/io_demo/iodemo_analyzer.py -d $(workspace)/${{ parameters.name }} --duration ${{ parameters.duration }}
+    python /hpc/noarch/git_projects/hpc-mtt-conf/scripts/iodemo_analyzer.py --branch $(Build.SourceBranchName) -d $(workspace)/${{ parameters.name }} --duration ${{ parameters.duration }} -t 3
```
Comment (yosefe) [inline buildlib/pr/io_demo/az-stage-io-demo.yaml:63]: manager_script_dir ?
what is "-t 3" ?

File: buildlib/pr/io_demo/io-demo.yml [hunk 3]
```diff
@@ -30,7 +30,7 @@ jobs:
 
     pool:
       name: MLNX
-      demands: ${{ parameters.demands }}
+      demands: ucx_iodemo_build -equals yes
```
Comment (yosefe) [inline buildlib/pr/io_demo/io-demo.yml:33]: 1. why need a special label here? 
2. maybe build on docker?

File: buildlib/pr/io_demo/io-demo.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 parameters:
   - name: demands
     type: string
-    default: "ucx_iodemo -equals yes"
+    default: "ucx_iodemo_rain -equals yes"
```
Comment (yosefe) [inline buildlib/pr/io_demo/io-demo.yml:4]: maybe "ucx_roce_iodemo" label?

File: buildlib/pr/io_demo/az-stage-io-demo.yaml (hunk 2 above)
Comment (avildema) [reply buildlib/pr/io_demo/az-stage-io-demo.yaml:63]: threshold for 'w/o traffic'

File: buildlib/pr/io_demo/io-demo.yml (hunk 3 above)
Comment (avildema) [reply buildlib/pr/io_demo/io-demo.yml:33]: good idea, will try build on centos 7

File: buildlib/pr/io_demo/io-demo.yml (hunk 4 above)
Comment (avildema) [reply buildlib/pr/io_demo/io-demo.yml:4]: no no, it's for test on rain, will revert back to 'ucx_iodemo'

File: buildlib/az-network-corrupter.sh (hunk 1 above)
Comment (avildema) [reply buildlib/az-network-corrupter.sh:18]: switch port up/down on python , take from mtt , for avoid double support for a same scripts

File: buildlib/pr/io_demo/az-stage-io-demo.yaml (hunk 2 above)
Comment (yosefe) [reply buildlib/pr/io_demo/az-stage-io-demo.yaml:63]: in minutes?
what was it before this change?

File: buildlib/az-network-corrupter.sh (hunk 1 above)
Comment (yosefe) [reply buildlib/az-network-corrupter.sh:18]: can't just run
`${manager_script} -d ${manager_script_dir}/hosts --host $(hostname) -a on -i ${interface}`
why need "python" in the command line?

File: buildlib/pr/io_demo/az-stage-io-demo.yaml (hunk 2 above)
Comment (avildema) [reply buildlib/pr/io_demo/az-stage-io-demo.yaml]: was 3min by default, but in mtt repo threshold  is 1min by default

File: buildlib/az-network-corrupter.sh (hunk 1 above)
Comment (avildema) [reply buildlib/az-network-corrupter.sh:18]: this is file no executable

Comment (yosefe) [reply buildlib/az-network-corrupter.sh:18]: can you make it executable?

Comment (avildema) [reply buildlib/az-network-corrupter.sh]: done

File: buildlib/pr/io_demo/io-demo.yml (hunk 3 above)
Comment (avildema) [reply buildlib/pr/io_demo/io-demo.yml]: done

**Comment by yosefe** [comment]
@avildema can you pls add also `export UCX_RNDV_THRESH=4k` for the io_demo testing?

**Review by yosefe — APPROVED** [review]
pls squash

**Comment by yosefe** [comment]
merge issue: using wrong label for io_demo
https://github.com/openucx/ucx/compare/f57933e2cbf4f5928d2e98b5ec349c826f892d03..2c28825a9bf37746837f10931b4e71a1d5c2f4a4#diff-6d64434f23d8554a945d62fffcb37f7e53fbcd4816a30a99180b38479c0cca65

**Comment by yosefe** [comment]
@avildema is it possible there are docker tests running on rain-04?

**Comment by avildema** [comment]
> merge issue: using wrong label for io_demo
> https://github.com/openucx/ucx/compare/f57933e2cbf4f5928d2e98b5ec349c826f892d03..2c28825a9bf37746837f10931b4e71a1d5c2f4a4#diff-6d64434f23d8554a945d62fffcb37f7e53fbcd4816a30a99180b38479c0cca65
> 
> need to use "ucx_iodemo_rain"

It is correct, rain have this label, after merge will remove ucx_iodemo from sputnik

**Comment by avildema** [comment]
> @avildema is it possible there are docker tests running on rain-04?

currently - NO, rain 03-04 don't have docker, if needed I will install

**Review by yosefe — APPROVED** [review]

**Comment by yosefe** [comment]
> currently - NO, rain 03-04 don't have docker, if needed I will install

No need, asked because saw suspicious failure

**Pull request merged**